python_modules/airline-demo/airline_demo/pipelines.py

```diff
@@ -138,19 +67,22 @@ def _db_load(data_frame, table_name, resources):
         lambda info: ExecutionContext.console_logging(
             log_level=logging.DEBUG,
             resources=AirlineDemoResources(
-                _create_spark_session_local(), # FIXME
-                _create_s3_session(),
-                _create_redshift_db_url(
+                create_spark_session_local(), # FIXME
+                create_s3_session(),
+                create_redshift_db_url(
                     info.config['redshift_username'],
                     info.config['redshift_password'],
                     info.config['redshift_hostname'],
                     info.config['redshift_db_name'],
                 ),
-                _create_redshift_engine(
-                    info.config['redshift_username'],
-                    info.config['redshift_password'],
-                    info.config['redshift_hostname'],
-                    info.config['redshift_db_name'],
+                create_redshift_engine(
```

Comment:
is the url a property of the engine? if so we could eliminate the other resource

Reply:
it's not a public API on `Engine`, but the real issue here is that Spark wants a jdbc URL, and SQLAlchemy doesn't